proposals/esm-integration/EXAMPLES.md

```diff
@@ -31,7 +31,7 @@ Note: because these are snapshots of values, this does not maintain the live-bin
 ##### Function imports
 
 ```wasm
-;; main.wasm
```

Comment:
For these cases with `.wasm` and `.wat`, what do you think of writing it as `;; foo.wat --> foo.wasm`? I've seen this done in some documents such as the GC proposal JS API draft: https://github.com/WebAssembly/gc/blob/master/proposals/gc/MVP-JS.md#examples

Reply:
I think this is a great idea.